Code/Source/solver/mat_models.cpp

```diff
@@ -188,116 +360,120 @@ void get_pk2cc(const ComMod& com_mod, const CepMod& cep_mod, const dmnType& lDmn
   }
 
   // Electromechanics coupling - active strain
-  auto Fe  = F;
-  auto Fa = mat_id(nsd);
-  auto Fai = Fa;
-
-  if (cep_mod.cem.aStrain) {
-    actv_strain(com_mod, cep_mod, ya, nfd, fl, Fa);
-    Fai = mat_inv(Fa, nsd);
-    Fe = mat_mul(F, Fai);
-  }
-
-  double J = mat_det(Fe, nsd);
+  EigenMatrix<nsd> Fe  = F;
+  EigenMatrix<nsd> Fa = EigenMatrix<nsd>::Identity();
+  EigenMatrix<nsd> Fai = Fa;
+
+  // if (cep_mod.cem.aStrain) {
+  //   actv_strain(com_mod, cep_mod, ya, nfd, fl, Fa);
+  //   Fai = Fa.inverse();
+  //   Fe = F * Fai;
+  // }
```

Comment:
This code is for active strain. Active strain isn't currently tested, but if/when we want to test and use it, I think this commented code will be useful. IMO, it doesn't hurt to keep this commented code, and it could be helpful to a developer in the future.

Reply:
After discussing with Dave, I added a detailed comment explaining what this is, why it's commented out, and why I'm leaving it in for reference. Does this work?